### PR TITLE
Adding support for ServerTech Sentry4 PDU devices

### PR DESCRIPTION
**Pull request by jbradleynh**

### CODE REVIEW

File: generator/generator.yml [hunk 1]
```diff
@@ -108,6 +108,70 @@ modules:
...
+      - 1.3.6.1.4.1.1718.4.1.3.3   # st4InputCordMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.4.3   # st4LineMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.5.3   # st4PhaseMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.8.3   # st4OutletMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.9.3   # st4TempSensorMonitorTable
+      - 1.3.6.1.4.1.1718.4.1.14.3  # st4FanSensorMonitorTable
+
+    lookups:
+      - source_indexes:
+         - st4UnitIndex
```
Comment (brian-brazil) [inline generator/generator.yml:129]: You you keep these on one line like we have elsewhere in this file?

File: generator/generator.yml [hunk 2]
```diff
@@ -108,6 +108,70 @@ modules:
...
+        lookup: st4UnitName
+      - source_indexes:
+          - st4UnitIndex
+          - st4InputCordIndex
+        lookup: st4InputCordName
+        drop_source_indexes: true
```
Comment (brian-brazil) [inline generator/generator.yml:135]: Is st4InputCordName unqiue?

File: generator/generator.yml [hunk 3]
```diff
@@ -108,6 +108,70 @@ modules:
...
+        lookup: st4AdcSensorName
+        drop_source_indexes: true
+      - source_indexes:
+          - st4UnitIndex
+          - st4AdcSensorIndex
+          - st4FanSensorIndex
+        lookup: st4AdcSensorName
+        drop_source_indexes: true
+    overrides:
+      st4OutletID:
```
Comment (brian-brazil) [inline generator/generator.yml:166]: Why are you dropping these?

File: generator/generator.yml (hunk 1 above)
Comment (jbradleynh) [reply generator/generator.yml:129]: will clean this these up

File: generator/generator.yml (hunk 2 above)
Comment (jbradleynh) [reply generator/generator.yml:135]: yes they should be unique

File: generator/generator.yml (hunk 3 above)
Comment (jbradleynh) [reply generator/generator.yml:166]: they generate a lot a spurious output when combined with the other labels. I can try to recreate an example

Comment (jbradleynh) [reply generator/generator.yml]: these overrides are actually no longer needed and have been removed. They were left over from earlier tests where the OID to walk wasn't targeted properly.

File: generator/generator.yml (hunk 1 above)
Comment (jbradleynh) [reply generator/generator.yml]: refactored to single line to fit convention

**Comment by jbradleynh** [comment]
Unsure what is affecting the `ci/circleci: generator` check, it appears to be failing on the snmp diff for objections unrelated to this PR

**Comment by brian-brazil** [comment]
The CI is very particular. Make sure you've the latest tree, and the latest MIBs.

**Comment by adamcstephens** [comment]
Rebased and updated.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**